plugins/hydra_ray_launcher/setup.py

```diff
@@ -20,16 +20,18 @@
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
         "Operating System :: MacOS",
         "Operating System :: POSIX :: Linux",
     ],
     install_requires=[
-        "boto3==1.22.6",
-        "hydra-core>=1.1.2",
-        "ray[default]~=1.12",
-        "aiohttp==3.8.1",
-        "cloudpickle==2.0.0",
-        "pickle5==0.0.11; python_version < '3.8'",
+        "boto3>=1.28,<1.29",
```

Comment:
`boto3` is constantly updating minor versions, and this constraint may be too harsh. What about the following?

```suggestion
        "boto3>=1.28,<2",
```